rlberry/manager/utils.py

```diff
@@ -14,3 +16,58 @@ def create_database(db_file):
         connection.close()
         return True
     return False
+
+
+def tensorboard_folder_to_dataframe(path_to_tensorboard_data):
+    """
+    Function to convert 'tensorboard log' to 'Panda DataFrames'.\
+    To convert the 'tensorboard log', the input must be must be the path to "the parent folder of all the training log" (path_to_tensorboard_data), and the 'events.out.tfevents' files have to be in this kind of path :
+      < path_to_tensorboard_data/algo_name/n_simu/events.out.tfevents.xxxxx >.
+
+    The output format is a dictionary.
+        key = tag (type of data).\
+        value = Panda DataFrame with the following structure (4 column) :
+            "name" = algo_name.\
+            "n_simu" = n_simu (seed).\
+            "x" = step number.\
+            "y" = value of the data.
+
+    Parameters
+    ----------
+    path_to_tensorboard_data : path to the parent folder of the tensorboard's data.
+
+    Returns
+    -------
+    Dict : dict of Panda DataFrame (key = tag, value = Panda.DataFrame)
+    """
+    from tensorboard.backend.event_processing import event_accumulator
+
+    dataframe_by_tag = {}
+    for algo_name in os.listdir(path_to_tensorboard_data):
+        path_for_this_algo = os.path.join(path_to_tensorboard_data, algo_name)
+        if os.path.isdir(path_for_this_algo):
+            for seed in os.listdir(path_for_this_algo):
+                current_seed_path = os.path.join(path_for_this_algo, seed)
```

Comment:
Alternatively it would be nice to have this automatic folder exploration or let the user provide a list with all the current_seed_path. A parameter like
path_to_tensorboard_data: str or dict of list of str. If it is a str, it must be must be the path to "the parent folder of all the training log" and the individual metrics will be inferred from the path names. If it is a dict, the keys must be the metrics and the values must be a list of path for each seed.

Reply:
done !